Automattic/jetpack
# My Jetpack: improve My Jetpack link styles

My Jetpack: improve My Jetpack link styles

Fixes https://github.com/Automattic/jetpack/issues/23006

#### Changes proposed in this Pull Request:

* My Jetpack: improve My Jetpack link styles

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to My Jetpack overview
* Check the styles when hovering the links

https://user-images.githubusercontent.com/77539/155013120-008ed587-dab2-4b50-bfba-3be27664e47f.mov
